azkaban-common/src/main/java/azkaban/jobExecutor/ProcessJob.java

```diff
@@ -143,6 +175,15 @@ public void run() throws Exception {
     generateProperties(propFiles[1]);
   }
 
+  private boolean determineExecuteAsUser(Props sysProps, Props jobProps) {
+    boolean isExecuteAsUser = sysProps.getBoolean(EXECUTE_AS_USER, false);
+    // putting an override in case user needs to override. A temporary opening
+    if (jobProps.containsKey(EXECUTE_AS_USER_OVERRIDE))
+      isExecuteAsUser = jobProps.getBoolean(EXECUTE_AS_USER_OVERRIDE, false);
```

Comment:
Good point.

I did this because the 2 lines for EXECUTE_AS_USER_OVERRIDE (there are actually 2 different parameters in this method) is meant to be a short term loophole.  I would imaging those 2 lines be removed when the time comes to close this loophole.